files/usr/share/cinnamon/cinnamon-settings/bin/ExtensionCore.py

```diff
@@ -4,7 +4,7 @@
 import os
 import re
 import json
-import cgi
+import html
```

Comment:
This is already fixed on master, please rebase your PR and remove the last two commits.

Reply:
Done.